### ISSUE TITLE
export 'bFormSlider' was not found in 'vue-bootstrap-slider'

### ISSUE THREAD

**Issue by desthercz** [pimlie/vue-bootstrap-slider#9]
As the title says, i am experiencing error below after updating from version 1.3.2 to 1.4.0 version. I tried to downgrade and problem starts to occur in 1.3.3 and is not solved in current version 1.4.0.

**Error while webpack compilation**
`export 'bFormSlider' was not found in 'vue-bootstrap-slider'`

**I am using this code to import component:**
`import {bFormSlider} from 'vue-bootstrap-slider';`

**Workaround**
Downgrade to version 1.3.2

**Comment by pimlie** [comment]
My apologies, I removed that export indeed. I have released `v2.0.0` to indicate this breaking change, now you can use
```js
import bFormSlider from 'vue-bootstrap-slider/es/form-slider';
```
to import the component. Let me know if this works for you!

**Comment by desthercz** [comment]
Thank you, that works fine.
